python/lsst/qserv/tests/benchmark.py

```diff
@@ -177,6 +174,13 @@ def _parseFile(self, qF, withQserv):
             elif withQserv and line.startswith("-- withQserv"):
                 # strip the "-- withQserv" text
                 qText.append(line[13:])
+            elif line.endswith("-- noQserv"):
+                if withQserv:
+                    # skip this line
+                    pass
+                else:
+                    # strip the "-- noQserv" text
+                    qText.append(line[:-10])
```

Comment:
This replacement does not do exactly the same thing. Original regex search allowed comments like:

    SOME SQL -- noQserv (some comment here)

while this replacement does not allow anything after `noQserv`. Not sure if this matters, I presume that you checked that there is no comments like above in our existing tests.